azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,287 @@
+
+```Bash
+pip install azure-cognitiveservices-vision-computervision
+```
+
+## Authentication
+
+Once you create your Computer Vision resource, you need its **region**, and one of its **account keys** to instantiate the client object.
+
+Use these values when you create the instance of the [ComputerVisionAPI][ref_computervisionclient]. 
+
+### Get credentials
+
+Use the Azure CLI snippet below to populate two environment variables with the Computer Vision account URI and one of its key (you can also find these values in the Azure portal). The snippet is formatted for the Bash shell.
+
+```Bash
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+export ACCOUNT_REGION=$(az cognitiveservices account show \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --query location \
+    --output tsv)
+
+export ACCOUNT_KEY=$(az cognitiveservices account keys list \
+    --name $ACCT_NAME \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --query key1 \
+    --output tsv)
+```
+
+### Create client
+
+Once you've populated the `ACCOUNT_REGION` and `ACCOUNT_KEY` environment variables, you can create the [ComputerVisionAPI][ref_computervisionclient] client.
+
+```Python
+from azure.cognitiveservices.vision.computervision import ComputerVisionAPI
+from azure.cognitiveservices.vision.computervision.models import VisualFeatureTypes
+from msrest.authentication import CognitiveServicesCredentials
+
+import os
+region = os.environ['ACCOUNT_REGION']
+key = os.environ['ACCOUNT_KEY']
+
+credentials = CognitiveServicesCredentials(key)
+client = ComputerVisionAPI(region, credentials)
+```
+
+## Usage
+
+Once you've initialized a [ComputerVisionAPI][ref_computervisionclient] client, you can:
+
+* Analyze an image: You can analyze an iimage for certain features such as faces, colors, tags.   
+* Moderate images in content: An image can be analyzed for adult content.
+* Extract text from an image: An image can be scanned for text. In order to complete the analysis, Computer Vision may correct rotation before analysis. Computer Vision supports 25 languages, and automatically detects the language of extracted text.
+
+For more information about these resources, see [Working with Azure computer vision][computervision_resources].
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Computer Vision tasks, including:
+
+* [Analyze an image](#analyze-an-image)
+* [Analyze an image by domain](#analyze-an-image-by-domain)
+* [Get text description of an image](#get-text-description-of-an-image)
+* [Generate thumbnail](#generate-thumbnail)
+* [Get subject domain list](#get-subject-domain-list)
+
+### Analyze an image
+
+You can analyze an image for certain features. Use the `visual_features` property to set the types of analysis to perform on the image. Common values are `VisualFeatureTypes.tags` and `VisualFeatureTypes.description`.
```

Comment:
You might already have this on the TODO list, but these code-fenced types should be deep links into the API reference.